Faucet: Add log when importing faucet signer account

ziogaschr opened the pull request:
This PR adds just a log when the signer account is being imported, as some previous log messages (attached image) might be misleading, and the user might feel that account import failed.

![image](https://user-images.githubusercontent.com/802700/102818818-0ecac280-43db-11eb-9e2f-356b6714e4b0.png)

ziogaschr committed:
main: add log when importing faucet signer account

meowsbits reviewed (approved):
IMO it might be easier to read as `} else if err == nil {` which is the same logical case.

ziogaschr committed:
main: change if clause for readability

thx to feedback at https://github.com/etclabscore/core-geth/pull/261#pullrequestreview-556994714
